src/runtime/schema/schema.ts

```diff
@@ -116,21 +116,20 @@ export function create({
       isSchemaEmpty: () => {
         return __types.length === 0
       },
-      makeSchema: async devModeLayout => {
+      makeSchema: async () => {
         const nexusSchemaConfig = createNexusSchemaConfig(
           plugins,
           state.settings
         )
         const { schema, missingTypes, typegenConfig } = await makeSchema(
           nexusSchemaConfig
         )
-        if (
-          process.env.NEXUS_STAGE === 'dev' &&
-          nexusSchemaConfig.shouldGenerateArtifacts === true
-        ) {
+        if (nexusSchemaConfig.shouldGenerateArtifacts === true) {
```

Comment:
Now typegen is not really part of the API, but totally internal. We can change if we ever need, but this is simpler for now.

Reply:
API being, here, internal. The `makeSchema` function. We don't pass down anymore.